utils.go

```diff
@@ -18,3 +19,11 @@ func WriteTOML(path string, data interface{}) error {
 	defer f.Close()
 	return toml.NewEncoder(f).Encode(data)
 }
+
+func BuildpackIdToDir(id string) string {
```

Comment:
Could we call this `buildpackIDToDir`?
I don't think it needs to be exported, and see [here](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).

Reply:
fixed in d0a7f80